python/ironcore-alloy/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 # using https://hatch.pypa.io/latest/
 # hatch build -t wheel
-# hatch run test
-# hatch run coverage
+# hatch run test:test
+# hatch run test:coverage
 # hatch shell
 [build-system]
```

Comment:
Looks like the opening `[` is still hanging out?

Reply:
I didn't touch the build-system line, it's just doing weird syntax highlighting